### PR TITLE
Dev Prod Sync

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Dev Prod Sync.

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/calling-java-code-from-ballerina.md [hunk 1]
```diff
@@ -1049,7 +1049,7 @@ function appendStringBuffer(handle sbObj, handle sb) returns handle = @java:Meth
 ```
 
 ### Java Exceptions as Ballerina Errors
-A function call in Ballerina may complete abruptly by returning an error or by raising a panic. Panics are rare in Ballerina. The best practise is to handle errors in your normal control flow. Raising a panic is similar to throwing a Java exception. The `trap` action will stop a panic and give you the control back in Ballerina and the `try-catch` statement does the same in Java. 
+A function call in Ballerina may complete abruptly by returning an error or by raising a panic. Panics are rare in Ballerina. The best practise is to handle errors in your normal control flow. Raising a panic is similar to throwing a Java exception. The `trap` action will stop a panic and give you the control back in Ballerina and the `try-catch` statement does the same in Java.
```
Comment (Samallama) [inline learn/calling-java-code-from-ballerina.md:1052]:
```suggestion
A function call in Ballerina may complete abruptly by returning an error or by raising a panic. Panics are rare in Ballerina. The best practice is to handle errors in your normal control flow. Raising a panic is similar to throwing a Java exception. The `trap` action will stop a panic and give you the control back in Ballerina and the `try-catch` statement does the same in Java.
```

File: learn/calling-java-code-from-ballerina.md [hunk 2]
```diff
@@ -1103,7 +1103,7 @@ public function main() {
 } 
 ```
 #### Java Checked Exceptions
-Let’s see how you can call a Java method that throws a checked exception. As illustrated in the following example, the corresponding Ballerina function should have the `error` type as part of it’s return type. 
+Let’s see how you can call a Java method that throws a checked exception. As illustrated in the following example, the corresponding Ballerina function should have the `error` type as part of it’s return type.
```
Comment (Samallama) [inline learn/calling-java-code-from-ballerina.md:1106]:
```suggestion
Let’s see how you can call a Java method that throws a checked exception. As illustrated in the following example, the corresponding Ballerina function should have the `error` type as part of its return type.
```

File: swan-lake/learn/deployment/docker.md [hunk 3]
```diff
@@ -466,7 +466,7 @@ This sample sends a greeting to the caller by getting the name from a text file.
 }
 ```
 
->**Note:** In addition to the above, if you want to copy driver files such as JDBC, you can create a Ballerina project, add following entires to its `Ballerina.toml` file, and change the path to the JDBC driver appropriately.
+>**Note:** In addition to the above, if you want to copy driver files such as JDBC, you can create a Ballerina project, add following entries to its `Ballerina.toml` file, and change the path to the JDBC driver appropriately.
```
Comment (Samallama) [inline swan-lake/learn/deployment/docker.md:469]:
```suggestion
>**Note:** In addition to the above, if you want to copy driver files such as JDBC, you can create a Ballerina project, add the following entries to its `Ballerina.toml` file, and change the path to the JDBC driver appropriately.
```

**Comment by praneesha** [comment]
Closing since this is invalid.

File: learn/calling-java-code-from-ballerina.md (hunk 1 above)
Comment (praneesha) [reply learn/calling-java-code-from-ballerina.md:1052]: Done in https://github.com/ballerina-platform/ballerina-dev-website/pull/1647.

File: learn/calling-java-code-from-ballerina.md (hunk 2 above)
Comment (praneesha) [reply learn/calling-java-code-from-ballerina.md:1106]: Done in https://github.com/ballerina-platform/ballerina-dev-website/pull/1647.

File: swan-lake/learn/deployment/docker.md (hunk 3 above)
Comment (praneesha) [reply swan-lake/learn/deployment/docker.md:469]: Done in .https://github.com/ballerina-platform/ballerina-dev-website/pull/1647

**Pull request closed without merging**